charts/argo-cd/values.yaml

```diff
@@ -1182,7 +1182,7 @@ redis:
     # -- Redis repository
     repository: public.ecr.aws/docker/library/redis
     # -- Redis tag
-    tag: 7.0.13-alpine
+    tag: 7.0.15-alpine
```

Comment:
Also please bump the image for `redis-ha` component on line 1372.

Reply:
fixed